pipes-and-filters/Pipes-And-Filters/Sender/src/main/java/Producer.java

```diff
@@ -1,11 +1,24 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 import java.io.IOException;
-import java.util.Scanner;
+import java.util.concurrent.Executors;
 import java.util.concurrent.TimeUnit;
 
 public class Producer {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws Exception {
```

Comment:
Fair point. The reason this does not do this originally is because we start with exercises where the producer has no loop, so it executes once. This is the first exercise where the producer runs in a loop. Now normally your producer would not be in a loop, unlike the consumer. So I guess the trade-off for me is whether adding the loop here removes the highlight of the fact that the consumer runs in a loop. Will ponder